### ISSUE TITLE
Request: Please give option to turn off lock screen flashlight and camera controls

### ISSUE THREAD

**Issue by jonathanzzero** [duraidabdul/Neptune#114]
Hello, 

I just wanted to make a request for the next version of Neptune. If possible, can there be an option to turn off the flashlight and camera toggle on the home screen? When notifications get in the way it doesn’t look good (at least to me). So if you can, that would be awesome. Besides that, great plugin man!

**Comment by eighteenspaces** [comment]
Yes please! This is my most wanted feature, as the toggles interfere with ColorFlow 4 ([screenshot](https://cdn.discordapp.com/attachments/313939755521933313/558175164773302283/image0.png) vs [how it normally looks](https://cdn.discordapp.com/attachments/313939755521933313/558177192543125534/image0.png)). It makes some elements far too small. It's bearable, but not ideal.

But honestly, that's my only gripe. Other than that, the tweak has performed perfectly.
